### PR TITLE
Add wallet list command

### LINKED ISSUE #108: Add Wallet List Command

### PR DESCRIPTION
**Pull request by gracegrape**

### CODE REVIEW

File: src/main/java/seedu/address/model/Wallet.java [hunk 1]
```diff
@@ -192,6 +192,17 @@ public Budget getBudget(Month month, Year year) {
     @Override
     public ObservableList<Transaction> getTransactionList() {
         ObservableList<Transaction> transactions = FXCollections.observableArrayList();
+
+        /*
```
Comment (joel-lim) [inline src/main/java/seedu/address/model/Wallet.java]: Commented out code

**Review by joel-lim — COMMENTED** [review]
Thanks for doing up a temp wallet UI!

File: src/main/java/seedu/address/model/Wallet.java (hunk 1 above)
Comment (gracegrape) [reply src/main/java/seedu/address/model/Wallet.java]: Thanks! Didn't catch this part.

File: src/main/java/seedu/address/ui/PersonListPanel.java [hunk 2]
```diff
@@ -34,6 +34,8 @@ public PersonListPanel(ObservableList<Person> personList) {
         protected void updateItem(Person person, boolean empty) {
             super.updateItem(person, empty);
 
+            logger.info("people have ");
```
Comment (syinyichen) [inline src/main/java/seedu/address/ui/PersonListPanel.java]: erm.. what is this for?

**Review by syinyichen — COMMENTED** [review]
They look good to me, thanks for adding the ui part for wallet!

File: src/main/java/seedu/address/ui/PersonListPanel.java (hunk 2 above)
Comment (gracegrape) [reply src/main/java/seedu/address/ui/PersonListPanel.java]: Sorry I was testing!! Thanks for noticing!

**Pull request merged**